src/adapters/watis.js

```diff
@@ -69,8 +70,40 @@ function findMatch(string) {
   }
   return parsed.replace(/\s+/g, ' ').trim();
 }
+async function addDescription(afkortingkje, description,event){
+  let descriptie = await Description.findOne({ afkorting:afkortingkje, betekening:description});
 
+  if(descriptie){
+    const response = {
+      message: 'Die descriptie betaat al',
+      channel: event.channel,
+      attachments: null,
+    };
+    Betty.emit('response', response);
+  }else{
+    let newthing = new Description({
+      afkorting:afkortingkje,
+      betekening:description
+    });
+    newthing.save();
```

Comment:
misschien wachten op het resultaat voor je aan de gebruiker gaat signaleren dat toevoegen aan de database gelukt is.
een save van mongoose kan met zowel callback als promise werken.